planetiler-core/src/test/java/com/onthegomap/planetiler/reader/ShapefileReaderTest.java

```diff
@@ -54,35 +55,29 @@ void testReadShapefileUnzipped() throws IOException {
   }
 
   private static void testReadShapefile(Path path) {
```

Comment:
Can we add another test that verifies it handles multiple shapefiles? It might be sufficient just to pass in `List.of(path, path)`? Alternatively we could add a test just for `SourceFeatureProcessor` that tests its handling with a stub implementation of a reader.